LemonUI/Scaleform/BaseScaleform.cs

```diff
@@ -142,6 +155,20 @@ private void CallFunctionBase(string function, params object[] parameters)
 
         #region Public Functions
 
+        /// <summary>
+        /// Check if the Scaleform has been loaded.
+        /// </summary>
+        public bool IsLoaded()
```

Comment:
This should also be in another PR.

Also, it should be in a Property instead of a Function.